static/docs/user-guide/howto/external-data-and-cache.md

```diff
@@ -0,0 +1,151 @@
+# Keep Data and Cache Outside the Project
+
+Sometimes we would like to keep the data and cache outside the project
+directory, so that we can search easily the code of the project.
+
+Keeping the data outside of the project, on a fixed absolute path, is also
+useful in case that there are several projects that use the same data.
+
+Another case when this would be desirable is if we want to keep the data on an
+[external drive](https://whatis.techtarget.com/definition/external-hard-drive)
+because they are too big to fit on our home directory, or for some other
+reasons.
+
+> A similar case is also when the data is on a
+> [network-attached storage (NAS)](https://searchstorage.techtarget.com/definition/network-attached-storage)
+> and is mounted through NFS, SSHFS, Samba, etc. However, in this case the data
+> is most probably used by a team of people, so make sure to check also the
+> guide about sharing data with a
+> [mounted DVC storage](/doc/user-guide/data-sharing/mounted-storage).
+
+## Make the data directory accessible
+
+Let's assume that the data of our project is located on `/data/project1/`. First
+we have to make sure that we can read and write the `/data/` directory:
+
+```dvc
+$ sudo chown <username>: -R /data/
```

Comment:
I think this is not right to provide chown, chmod and other commands to manage permissions. They are not actionable anyway. Exact commands you need to run depend on the case. Some general suggestions to check the permissions should be enough?